src/Project/EventListener/Insightly/RequestedActivationListener.php

```diff
@@ -84,13 +84,15 @@ public function handle(RequestedActivation $requestedActivation): void
         $this->insightlyClient->opportunities()->updateStage($insightlyOpportunityId, OpportunityStage::request());
         $this->insightlyClient->opportunities()->updateState($insightlyOpportunityId, OpportunityState::won());
 
+        $linkedContactId = $this->insightlyClient->opportunities()->getLinkedContactId($insightlyOpportunityId);
```

Comment:
Nit: This will probably fail or result in some kind of a null pointer exception if the contact link was removed by someone in Insightly? 🤔

Reply:
This can indeed happen. I will log an error message with an early return statement to avoid crashing.
I will also check the other new listeners for potential issues.